src/app/docs-pages/popup-global-doc/popup-global-doc.component.html

```diff
@@ -8,13 +8,22 @@ <h1 class="doc-title">Popup global service
     size of an overlay.
     The overlay will appear in the middle of the screen. To close the overlay use the detach method. <br />
   </p>
-  <p class="doc-info"> 
-    <b>
-      Typical use: Displaying some kind of information without any actions. 
-      The popup is more generic and made for use in non-specific scenarios, but can also be customized if needed. 
-    </b>
-  </p>
 
+  <div class="doc-info-use">
+    <h2><i class="fal fa-check-circle doc-info-use-icons icon-green"></i> <b>When to use:</b></h2>
+    <p>
+      Dialogs are used now that the user's attention must be directed to important information. 
+      The global popup should be used when you want to display information without any actions. 
```

Comment:
nei, global er ment for actions også

Reply:
Teksten sier ikke noe om når global skal brukes fremfor connected

Reply:
"The global popup should be used when you want to display information or actions as a popup that are not directly connected to a visual element on the page"